Value_converter/utilities.py

```diff
@@ -0,0 +1,10 @@
+
+#check if float
+def is_float(value):
```

Comment:
Questa e' un po' un casino e python non essendo fortemente tipato non aiuta.
Il nome `is_float` suggerisce che la funzione ritorni un booleano, o comunque qualcosa che mi indichi se il valora in input e' un float o no. Qui invece fai una conversione e in base al caso ritorni valori di tipi diversi.